package.json

```diff
@@ -14,7 +14,7 @@
         "@babel/plugin-transform-destructuring": "^7.20.2",
         "eslint": "^5.16.0",
         "mini-css-extract-plugin": "^1.6.2",
-        "node-sass": "^4.14.1",
+        "node-sass": "^8.0.0",
```

Comment:
We cannot do it because node-sass requires node v 19
...
![image](https://user-images.githubusercontent.com/52750352/204242281-4af5f508-c012-4c5f-ac1c-73f53cb66e4d.png)